<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QWidget" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="ql_period">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>120</horstretch>
        <verstretch>40</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Количество месяцев</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_6months">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>6 мес</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_9months">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>9 мес</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_12months">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>12 мес</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="ql_conditions">
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите условия</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="ql_rate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="ql_rate_value">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>382</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="ql_deposit_amount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>120</horstretch>
        <verstretch>40</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма вклада</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="deposit_amount">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_take">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оформить вклад</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="ql_payment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>120</horstretch>
        <verstretch>40</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выплата процентов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rb_payment_1">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>В конце срока</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_payment_2">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Link">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Accent">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>97</red>
              <green>197</green>
              <blue>84</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Каждый месяц</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
